smokey/src/main/java/frc/robot/Constants.java

```diff
@@ -73,6 +73,10 @@ public final class Constants {
     public static final double TrajectoryMaxAcceleration = 1;
     public static final double TrajectoryMaxVelocity = 1;
 
+    //*************************************** 
+    //For auto constants
+    public static final double snoutDepth = 5.25;
```

Comment:
Call this snoutDepthMeters and do the Units.inchesToMeters conversion here.

Reply:
done